contrib/babelfishpg_tsql/src/tsqlIface.cpp

```diff
@@ -2750,16 +2750,21 @@ rewriteBatchLevelStatement(
 	else if (ctx->create_or_alter_procedure())
```

Comment:
What about dml and ddl triggers? Don't we need the same fixes for them?

Reply:
Yes we need the same fix for ddl and dml triggers, fixed in next revision.